docs/nodejs.md

```diff
@@ -29,11 +29,18 @@ aa("init", {
 
 ## 3. Add `userToken`
 
-On the Node.js environment, unlike the browser environment, `userToken` must be specified when sending any event.
+On the Node.js environment, unlike the browser environment, a [user token](https://www.algolia.com/doc/guides/sending-events/concepts/usertoken) (`userToken` and/or `authenticatedUserToken`) must be specified when sending any event.
 
 ```js
+// Anonymous user ID
 aa("clickedObjectIDs", {
-  userToken: "USER_ID"
+  userToken: "ANONYMOUS_ID"
+  // ...
+});
+
+// Authenticated user ID
+aa("clickedObjectIDs", {
+  authenticatedUserToken: "USER_ID"
```

Comment:
I think we still want users to provide `userToken` here in addition to `authenticatedUserToken`.